src/util/UploadImages.js

```diff
@@ -0,0 +1,49 @@
+/* global Promise */
+'use strict';
+
+var fs = require('fs');
+var Eyes = require('@applitools/eyes-images').Eyes;
+
+var eyes = new Eyes();
+var ConsoleLogHandler = require('@applitools/eyes-images').ConsoleLogHandler;
+eyes.setLogHandler(new ConsoleLogHandler(true));
+eyes.setApiKey(process.env.APPLITOOLS_API_KEY);
```

Comment:
curious how this key be set in travis ?

Reply:
travis secure env vars can be used. It'll be encrypted. We already set okta API tokens in travis for running DevEx SDK tests.